src/scss/global.scss

```diff
@@ -1,14 +1,51 @@
+/* Accessibility Color Variables */
+
+$main-theme: #005eaa;
+$main-theme-dark: #004f8f;
+$main-theme-darker: #004175;
+$main-theme-light: #88c3ea;
+$main-theme-lighter: #c0e9ff;
+
+$gray-dark: #555555;
+$gray-darker: #333333;
+$gray-light: #e0e0e0;
+
+$primary-blue: #196eb3;
+$primary-brown: #705043;
+$primary-cyan: #007b91;
+$primary-green: #497d0c;
+$primary-indigo: #26418f;
+$primary-orange: #bb4d00;
+$primary-pink: #af4448;
+$primary-purple: #712177;
+$primary-slate: #29434e;
+$primary-teal: #00695c;
+
+$tertiary-amber: #ffecb3;
+$tertiary-blue: #c0e9ff;
+$tertiary-brown: #decec6;
+$tertiary-cyan: #cce5e9;
+$tertiary-green: #dcedc8;
+$tertiary-indigo: #dee8ff;
+$tertiary-orange: #ffe97d;
+$tertiary-pink: #ffc2c2;
+$tertiary-purple: #e3d3e4;
+$tertiary-slate: #b6c6d2;
+$tertiary-teal: #ceece7;
+
 *,
 html,
 body {
-  --main-theme: #005eaa;
-  --main-light: #88c3ea;
-  --main-lighter: #c0e9ff;
-  --indigo: #26418f;
+  --main-theme: $main-theme;
+  --main-light: $main-theme-light;
+  --main-lighter: $main-theme-lighter;
+  --indigo: $primary-indigo;
   --heading-xl: 3.1rem;
   font-size: 10px;
```

Comment:
Change this to `font-size: 17px;`
The base font-size for CDC styles is 17px. We will want to set that here on the body tag css so it is applied across subsequent rem declarations.

Reply:
Can we discuss this?  Why 17px?  We set to 10px to make it easier for us to set rem values.

Reply:
Also, note that rem is set relative to the base size, so a rem of 1 would be equal to whatever base size is set (in this case 17px)

Reply:
Let's do this in a follow on ticket.  Seems like a big refactor.